e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/submit-deal-cancellation-in-past.spec.js

```diff
@@ -105,6 +107,16 @@ context('Deal cancellation - effective from date in past', () => {
         const row = facilitiesPage.facilitiesTable.row(facilityId);
         cy.assertText(row.facilityStage(), TFM_FACILITY_STAGE.RISK_EXPIRED);
       });
+
+      it('should add an entry on the activity and comments page', () => {
+        caseSubNavigation.activityLink().click();
+
+        activitiesPage.activitiesTimeline().contains('Deal stage:');
+        activitiesPage.activitiesTimeline().contains('Cancelled');
+        activitiesPage.activitiesTimeline().contains(`Bank request date: ${format(today.date, 'd MMMM yyyy')}`);
```

Comment:
Here you can use `Bank request date: ${today.d_MMMM_yyyy}`

Reply:
So true, thats the whole point of this